apps/meteor/client/views/admin/ABAC/AdminABACLogs.tsx

```diff
@@ -0,0 +1,179 @@
+import { useMemo, useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import GenericNoResults from '../../../components/GenericNoResults';
+import {
+	GenericTable,
+	GenericTableBody,
+	GenericTableCell,
+	GenericTableHeader,
+	GenericTableHeaderCell,
+	GenericTableRow,
+} from '../../../components/GenericTable';
+import { usePagination } from '../../../components/GenericTable/hooks/usePagination';
+import { ABACQueryKeys } from '../../../lib/queryKeys';
+import DateRangePicker from '../moderation/helpers/DateRangePicker';
+
+const AdminABACLogs = () => {
+	const { t } = useTranslation();
+
+	const [startDate, setStartDate] = useState<string>(new Date().toISOString().split('T')[0]);
+	const [endDate, setEndDate] = useState<string>(new Date().toISOString().split('T')[0]);
+
+	const { current, itemsPerPage, setItemsPerPage, setCurrent, ...paginationProps } = usePagination();
+	const getLogs = useEndpoint('GET', '/v1/abac/audit');
+	const query = useMemo(
+		() => ({
+			...(startDate && { start: new Date(`${startDate}T00:00:00.000Z`).toISOString() }),
+			...(endDate && { end: new Date(`${endDate}T23:59:59.999Z`).toISOString() }),
+			offset: current,
+			count: itemsPerPage,
+		}),
+		[current, itemsPerPage, startDate, endDate],
+	);
+
+	// Whenever the user changes the filter or the text, reset the pagination to the first page
+	useEffect(() => {
+		setCurrent(0);
+	}, [startDate, endDate, setCurrent]);
+
+	const { data, isLoading } = useQuery({
+		queryKey: ABACQueryKeys.logs.logsList(query),
+		queryFn: () => getLogs(query),
+	});
+
+	const getActionLabel = (action?: AbacAttributeDefinitionChangeType | null) => {
+		switch (action) {
+			case 'created':
+				return t('Created');
+			case 'updated':
+				return t('Updated');
+			case 'deleted':
+				return t('Deleted');
+			case 'all-deleted':
+				return t('ABAC_All_Attributes_deleted');
+			case 'key-removed':
+				return t('ABAC_Key_removed');
+			case 'key-renamed':
+				return t('ABAC_Key_renamed');
+			case 'value-removed':
+				return t('ABAC_Value_removed');
+			case 'key-added':
+				return t('ABAC_Key_added');
+			case 'key-updated':
+				return t('ABAC_Key_updated');
+			default:
+				return '';
+		}
+	};
+
+	const getEventInfo = (
+		event: Serialized<OperationResult<'GET', '/v1/abac/audit'>>['events'][number],
+	): { element: string; userAvatar: ReactNode; user: string; name: string; action: string; timestamp: string } => {
+		if (event.t === 'abac.attribute.changed') {
+			return {
+				element: t('ABAC_Room_Attribute'),
+				userAvatar: event.actor?.type === 'user' ? <UserAvatar size='x28' userId={event.actor._id} /> : null,
+				user: event.actor?.type === 'user' ? event.actor.username : t('System'),
+				name: event.data?.find((item) => item.key === 'attributeKey')?.value ?? '',
+				action: getActionLabel(event.data?.find((item) => item.key === 'change')?.value),
+				timestamp: new Date(event.ts).toLocaleString(),
```

Comment:
Just convert `event.ts` as a `Date` object and use a formatting function at the JSX below.